lib/questions.rb

```diff
@@ -187,12 +223,14 @@ def word_count_a_file(file_path)
 # called call_method_from_string('foobar')
 # the method foobar should be invoked
 def call_method_from_string(str_method)
+  call(str_method)
 end
 
 # return true if the date is a uk bank holiday for 2014
 # the list of bank holidays is here:
 # https://www.gov.uk/bank-holidays
 def is_a_2014_bank_holiday?(date)
+  [Time.parse('1/1/2014'),Time.parse('18/4/2014'),Time.parse('21/4/2014'),Time.parse('5/5/2014'),Time.parse('26/5/2014'),Time.parse('25/8/2014'),Time.parse('25/12/2014'),Time.parse('26/12/2014')].any?{|e|e==date}
```

Comment:
Line is too long. [212/80]<br>Space missing after comma.<br>Space after closing `|` missing.<br>Surrounding space missing for operator '=='.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.